backend/src/Models/VoterRolls.ts

```diff
@@ -0,0 +1,125 @@
+import { VoterRoll } from '../../../domain_model/VoterRoll';
+var format = require('pg-format');
+class VoterRollDB {
+
+    _postgresClient;
+    _tableName: string;
+
+    constructor(client: any, tableName: string) {
+        this._postgresClient = client;
+        this._tableName = tableName;
+    }
+
+    init(): Promise<VoterRollDB> {
+        console.log("VoterRollDB.init");
+        var query = `
+        CREATE TABLE IF NOT EXISTS ${this._tableName} (
```

Comment:
I want to make sure I'm understanding the usage correctly, is this example correct?

A user wants to vote, so we query the voter roll db

```
SELECT * FROM table WHERE voter_id=voter
```

and then we allow the user to vote if there's a matching entry, and submitted is false?

Reply:
Depends on the election settings. In a closed election correct. In an open election we'd still allow them to vote if there isn't a matching entry and then create one when the ballot is submitted.